src/bridge/cockpitpeer.c

```diff
@@ -51,6 +53,12 @@ struct _CockpitPeer {
   /* Authorizations going on */
   GHashTable *authorizes;
 
+  /* authorize types we will reply to */
+  GHashTable *authorize_values;
```

Comment:
Will do this as a follow up. I actually think the use of "authorize" is sound, it may just need to use a key prompting mechanism.